tools/launch_benchmark_suite_scheduler_slurm.py

```diff
@@ -0,0 +1,82 @@
+import sys
+
+import submitit
+from fvcore.common.file_io import PathManager
+from vissl.config.attr_dict import AttrDict
+from vissl.engines.benchmark_suite_scheduler import BenchmarkSuiteScheduler
+from vissl.utils.io import load_file
+
+
+# Default slurm options to pass to the executor.
+_DEFAULT_SLURM_OPTIONS = {
+    "NAME": "vissl",
+    "COMMENT": "vissl evaluation job",
+    "PARTITION": "learnlab",
+    "CONSTRAINT": "",
```

Comment:
nit: remove this from default. I think we should let user explicitly decide the constraint and not make default assumption

Reply:
I think we should definitely make this overrideable, but I still think we should provide a sensible default.

Reply:
It's highly likely that we specify constraints like: "volta32g" to be able to leverage 32GB machines for most of our evaluations.

Reply:
This is for the eval monitoring script, not the evaluations.